content/docs/pipelines/sdk/client_usage.md

```diff
@@ -0,0 +1,43 @@
++++
+title = "Use Kubeflow Pipelines API Client"
+description = "KFP SDK version >= 0.2.5 is required"
+weight = 30
++++
+
+This page includes a quick example on how to create a pipeline and a pipeline version using KFP API client.
+
+
+## Before you start
+
+* Install the [Kubeflow Pipelines SDK](/docs/pipelines/sdk/install-sdk/). The following requires KFP SDK version equal to or greater than 0.2.5. You can test your installed version using
+```
+pip list | grep kfp
+```
+
+## Example 1: create a pipeline and then create a new version under it
+
+* First, we use KFP API client to create a pipeline from a local file.
```

Comment:
I would recommend replacing the listed with a paragraph that describes what the code does. Then, we can add a section about the variable replacements after the example. If the user does not have a pipeline, is there a sample pipeline yaml that you would recommend that they use?

Suggested replacement:

The following example demonstrates how to use the Kubeflow Pipelines SDK to create a pipeline and a pipeline version.

Reply:
I put suggested replacement at the top of the page.